aces/Loris
# [Imaging Browser] Updated module help text

## Brief summary of changes

New copy for Imaging Browser module help text
#### Testing instructions (if applicable)

1. Edit copy if necessary
2. Check out screenshots of rendering below

#### Links to related tickets (GitHub, Redmine, ...)

*https://github.com/aces/Loris/issues/5576

![Screenshot 2019-11-12 22 48 02](https://user-images.githubusercontent.com/37813606/68731563-81455e00-059e-11ea-8ee8-050d38aa2d1c.png)
![Screenshot 2019-11-12 22 47 23](https://user-images.githubusercontent.com/37813606/68731564-81455e00-059e-11ea-9c3c-871e7584cb36.png)
